camunda-formio-tasklist-vue/src/components/TaskList.vue

```diff
@@ -111,12 +115,19 @@
                 </span>
               </b-col>
               <b-col cols="3" class="align-self-center">
-                <div id="groups" v-b-modal.AddGroupModal class="group-align word-break">
+                <div
+                  id="groups"
+                  v-b-modal.AddGroupModal
+                  class="group-align word-break"
+                >
                   <i class="fa fa-th mr-1"></i>
-                  <span class="cft-group-align cft-word-break" v-if="groupListNames">
+                  <span
+                    class="cft-group-align cft-word-break"
+                    v-if="groupListNames"
+                  >
                     {{ String(groupListNames) }}
```

Comment:
why is this filter added?

Reply:
Nope. String is used to convert Array for variables to comma separated variables form